src/NavBar/NavBar.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./NavBar.scss";
+import { NavLink } from "react-router-dom";
+import statsIcon from "../assets/bar-chart.svg";
+import recipesIcon from "../assets/chef.svg";
+import inventoryIcon from "../assets/clipboard.svg";
+import groceryIcon from "../assets/groceries.svg";
+import userIcon from "../assets/user.svg";
+
+const NavBar = () => {
+  return (
+    <>
+      <nav>
+        <div className="vertical point">
+          <img alt="statistics" src={userIcon} />
```

Comment:
Because we have text, the best practices for the alt text would be to leave alt="". This is because this is an icon / decorative image and not actually adding to the content of the page, and again this is because we have the text there. If we only had icons and no text, then we would want to fill in the alt text with the text for each. ex: alt="profile". My partner does SEO and I consulted him on this. This will be the same advice for all of the icon images in this section. Also I love the icons you found, great choices!

Reply:
That is very interesting. Thank you for pointing that out. I'll make the change for all the icons to alt="".